products/oceanic-current-fronts-world-ocean-circulation-WE/collection.json

```diff
@@ -46,7 +46,7 @@
     },
     {
       "rel": "self",
-      "href": "https://esa-earthcode.github.io/open-science-catalog-metadata/products/oceanic-current-fronts-world-ocean-circulation/collection.json",
```

Comment:
why the project link changed with the prefix -WE? is this something you would like to change?